packages/web/src/executors/rollup/lib/update-package-json.spec.ts

```diff
@@ -2,10 +2,6 @@ import { updatePackageJson } from './update-package-json';
 import * as utils from 'nx/src/utils/fileutils';
 import { PackageJson } from 'nx/src/utils/package-json';
 
-jest.mock('nx/src/utils/fileutils', () => ({
-  writeJsonFile: () => {},
-}));
```

Comment:
If this is being removed, we should update the spies to mock its implementation s.t. files are not written to disk

Reply:
I mocked `fs` with `memfs.fs`